graphql_api/query/embedding.py

```diff
@@ -49,3 +52,27 @@ def resolve_project_tokenization(
                 progress=-1,
             )
         return tokenization.get_record_tokenization_task(project_id)
+
+    def resolve_embedding_platforms(self, info) -> List[EmbeddingPlatform]:
+        return [
```

Comment:
wouldn't this be something for the embedding repo that is passed through like the suggestions?

Reply:
done